projects/demo-playwright/tests/kit/carousel/carousel.spec.ts

```diff
@@ -0,0 +1,36 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`Carousel`, () => {
+    test(`default padding`, async ({page}) => {
+        await tuiGoto(page, `components/carousel/API`);
+        const documentationPage = new TuiDocumentationPagePO(page);
+        const example = documentationPage.apiPageExample;
+
+        await expect(example).toBeVisible();
+        await expect(example).toHaveScreenshot(`carousel-padding-default.png`);
+    });
+
+    test(`padding has zero value`, async ({page}) => {
+        await tuiGoto(page, `components/carousel/API?style.--tui-carousel-padding=0`);
+        const documentationPage = new TuiDocumentationPagePO(page);
+        const example = documentationPage.apiPageExample;
+
+        await expect(example).toBeVisible();
+        await expect(example).toHaveScreenshot(`carousel-padding-0.png`);
+    });
+
+    test(`should show next item after drag`, async ({page}) => {
+        await tuiGoto(page, `components/carousel/API?draggable=true`);
+        const documentationPage = new TuiDocumentationPagePO(page);
+        const example = documentationPage.apiPageExample;
+        const itemExample = example.locator(`.t-item`).first();
+
+        await itemExample.dispatchEvent(`mousedown`);
+        await itemExample.dispatchEvent(`mousedown`, {x: 600, y: 100});
+        await itemExample.dispatchEvent(`mousedown`, {x: -0, y: 100});
+        await itemExample.dispatchEvent(`mouseup`);
```

Comment:
Should we use Playwright API ? 🤔 

https://playwright.dev/docs/input#dragging-manually

Reply:
We do use playwright native API. dispatchEvent is a method from playwright-core. 
I can not use <locator>.dragTo(<locator>) because the only one argument in signature can not be anything except Locator. And at the same time it's slider, so there is no appropriate Locator for this task.

Reply:
Ok, I rewrite with page.mouse